twistedchecker/functionaltests/comments.py

```diff
@@ -15,4 +15,25 @@
 #  x = x + 1
 
 # Make sure no error occur when checking an empty comment
-#  
\ No newline at end of file
+#
+
+"""
+#twisted checker things this is a comment and so
+1. Comments should begin with one whitespace
+2. The first letter of comment should be capitalized
+
+Line with an epydoc C{#} markup.
+
+This is particularly triggered with url fragments
+@see U{https://example.com/test#fragment}
+"""
+
+
+class SomeClass(object):
+    """
+    #twisted checker things this is a comment.
+
+    Line with an epydoc C{#} markup.
+
+    Also triggered with url fragments U{https://example.com/test#fragment}
+    """
```

Comment:
Could there be an extra line with a wrong comment after the docstring, to make sure that it works after the docstring checks have been turned off?